op-chain-ops/genesis/layer_two.go

```diff
@@ -46,3 +54,74 @@ func BuildL2DeveloperGenesis(config *DeployConfig, l1StartBlock *types.Block) (*
 
 	return db.Genesis(), nil
 }
+
+// BuildL2MainnetGenesis will build an L2 Genesis suitable for a Superchain mainnet that does not
+// require a pre-bedrock migration & supports optional governance token predeploy.
+func BuildL2MainnetGenesis(config *DeployConfig, l1StartBlock *types.Block) (*core.Genesis, error) {
```

Comment:
I don't love the name of this function, ideally this is what the `op-node` uses for its genesis command for devnets long term, i think its ok for now

Reply:
I called it a "mainnet" config since I don't prefund any accounts like the dev-genesis function was doing.  We could add a bool for prefunding dev accounts here, or make it a config option?

Reply:
I think its fine as it is, we don't need to worry about it too much right now